float64[] esdf_buffer
int32 grid_size_x
int32 grid_size_y
int32 grid_size_z
float64 resolution
float64 min_x
float64 min_y
float64 min_z
